modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,6 +19,8 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {ProductShortName} fixed issues
 
+* link:https://issues.redhat.com/browse/OSSM-419[OSSM-419] Namespaces with similar names will all show in Kiali namespace list, even though namespaces may not be defined in Service Mesh Member Role.
```

Comment:
It looks like the convention for including issue links as part of a New Feature is to put it at the end of the description and format it as "See [issue link] for more information.

https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html#ocp-4-7-rhcos